### PR TITLE
Feature find nearby serice

### PR DESCRIPTION
**Pull request by kamilsdz**

### CODE REVIEW

File: app/controllers/pages_controller.rb [hunk 1]
```diff
@@ -11,5 +11,10 @@ def dashboard
     @data[:maintenances] = current_user.maintenances.most_recent(5)
     @data[:mileages] = current_user.mileages.most_recent(5).decorate
     @data[:reminders] = current_user.reminders.most_recent(5)
+    @data[:closest_service] = find_nearby['car_repair']
```
Comment (fab-ian) [inline app/controllers/pages_controller.rb]: Why ``` find_nearby['car_repair'] ```? Shouldn't be it ``` find_nearby ```

File: app/services/find_nearby.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class FindNearby
+  AVAILABLE_PLACES = %w[car_dealer car_rental car_wash car_repair gas_station parking].freeze
+  def find_places(latitude, longitude)
+    places = {}
```
Comment (fab-ian) [inline app/services/find_nearby.rb]: Use 'tap'. Looks much cleaner. Alicja learned me this :)
```
{}.tap do |places|
    AVAILABLE_PLACES.each do |place|
      places[place] = GoogleApi.find_objects_nearby(latitude, longitude, place)['results']
    end
end
```

File: app/services/find_nearby.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class FindNearby
+  AVAILABLE_PLACES = %w[car_dealer car_rental car_wash car_repair gas_station parking].freeze
```
Comment (fab-ian) [inline app/services/find_nearby.rb]: AVAILABLE_PLACES? Maybe AVAILABLE_SERVICES?

File: app/views/pages/dashboard_blocks/_closest_service.html.haml [hunk 4]
```diff
@@ -1,4 +1,12 @@
...
 
-#{ data[:closest_service] }
+%table.table.table-hover
+  %tbody
+    - data[:closest_service].first(3).each do |service|
+      %tr
+        %td #{service['name']}
+        %td #{service['vicinity']}
+        %td
+          %button.btn.btn-light{:type => "button"} #{I18n.t('see')}
```
Comment (fab-ian) [inline app/views/pages/dashboard_blocks/_closest_service.html.haml]: What does it do? There isn't linked anywhere.

File: app/controllers/pages_controller.rb (hunk 1 above)
Comment (kamilsdz) [reply app/controllers/pages_controller.rb]: Yep, it's not too pretty. I prepared find_nearby service to take few "places" from PLACES API (car_dealer, car_rental, car_wash, etc.) and returns it in hash. So to add new place to view we can simply add new attribute to the controller.

File: app/services/find_nearby.rb (hunk 3 above)
Comment (kamilsdz) [reply app/services/find_nearby.rb]: Namespace is based on GOOGLE PLACES API: https://developers.google.com/places/web-service/search

File: app/services/find_nearby.rb (hunk 2 above)
Comment (kamilsdz) [reply app/services/find_nearby.rb]: Good tip Fabian! :)

File: app/views/pages/dashboard_blocks/_closest_service.html.haml (hunk 4 above)
Comment (kamilsdz) [reply app/views/pages/dashboard_blocks/_closest_service.html.haml]: At this moment anywhere:) Our new task in trello will be responsible for that. We can use method get_place_details from google_api module to present details about place. Here you can look at sample response: https://developers.google.com/places/web-service/details

Comment (fab-ian) [reply app/views/pages/dashboard_blocks/_closest_service.html.haml]: ok :)

File: app/controllers/pages_controller.rb (hunk 1 above)
Comment (fab-ian) [reply app/controllers/pages_controller.rb]: There is above an unused ```  @data[:closest_service]  ``` declaration :)
Remove this.

File: app/services/find_nearby.rb (hunk 3 above)
Comment (fab-ian) [reply app/services/find_nearby.rb]: ok, AVAILABLE_TYPES based on : https://developers.google.com/places/web-service/supported_types

**Pull request merged**